frontend/public/files/ches.postman_collection.json

```diff
@@ -1,10 +1,364 @@
 {
 	"info": {
-		"_postman_id": "76db54d3-dfad-4970-9be4-d6b628f37633",
-		"name": "Common Hosted Email",
-		"description": "The NR Common Services CHES (Common Hosted Email Service) API",
+		"_postman_id": "4add4d38-2418-4df2-8e81-b68f05069606",
+		"name": "Common Hosted Email Service",
+		"description": "CHES - Powered by NodeMailer (a shared library)",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
-	"item": [],
+	"item": [
+		{
+			"name": "v1",
+			"item": [
+				{
+					"name": "Health",
+					"item": [
+						{
+							"name": "/health",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "http://localhost:3000/api/v1/health",
```

Comment:
Are we worried about the urls and tokens in the collection for this release?

Reply:
I meant to change the URLs to the prod CHES instance but forgot, I will do that.

Reply:
Fixed to proper URLs and removed expired token @parc-jason